onnxruntime/contrib_ops/cuda/fused_conv.cc

```diff
@@ -47,25 +47,25 @@
     const auto alpha = onnxruntime::cuda::Consts<CudaT>::One;
     const auto beta = onnxruntime::cuda::Consts<CudaT>::Zero;
     IAllocatorUniquePtr<void> workspace = Base::GetWorkSpace();
-    auto cudnn_status = cudnnConvolutionBiasActivationForward(Base::CudnnHandle(),
-                                                              &alpha,
-                                                              Base::s_.x_tensor,
-                                                              Base::s_.x_data,
-                                                              Base::s_.w_desc,
-                                                              Base::s_.w_data,
-                                                              Base::s_.conv_desc,
-                                                              Base::s_.algo,
-                                                              workspace.get(),
-                                                              Base::s_.workspace_bytes,
-                                                              has_z ? &alpha : &beta,
-                                                              has_z ? Base::s_.z_tensor : Base::s_.y_tensor,
-                                                              has_z ? Base::s_.z_data : Base::s_.y_data,
-                                                              Base::s_.b_tensor,
-                                                              has_b ? Base::s_.b_data : Base::s_.b_zero,
-                                                              activation_desc_,
-                                                              Base::s_.y_tensor,
-                                                              Base::s_.y_data);
-    if (CUDNN_STATUS_SUCCESS != cudnn_status) {
+
+    if (!has_b || CUDNN_STATUS_SUCCESS != cudnnConvolutionBiasActivationForward(Base::CudnnHandle(),
```

Comment:
> CUDNN_STATUS_SUCCESS != cudnnConvolutionBiasActivationForwar [](http://example.com/codeflow?start=17&length=61)

I am wondering what this failure mean and why we do not error out? At the very least this needs to be documented.

Reply:
Added error check and return on failure.